docs/deploy/deployment-manifest.md

```diff
@@ -141,7 +142,7 @@ An example manifest for the released Industrial-IoT IoT Edge modules included in
               "image": "mcr.microsoft.com/iotedge/opc-twin:latest"
             }
           },
-          "opcpublisher": {
+          "publisher": {
```

Comment:
Why lose the "opc" again? I thought we agreed to keep it?

Reply:
I am using the same names here as we have in layered deployments, which are without `opc-` prefix.